mycelium-api/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "mycelium-api"
+version = "0.5.2"
+edition = "2021"
```

Comment:
This is missing the `license-file` and `readme` keys (both can link to the parent directory files, similar to other crates)